### DISCUSSION TITLE
Data table:  unit test cases issue

### DISCUSSION THREAD

**Discussion by ambashy** [primefaces/community#1190] [PrimeVue]
How to pass the mock data  in data table  using jest and @vue-test-util ?

**Comment by tugcekucukoglu** [comment]
As a starter, PrimeVue can be added as a plugin:

```
import PrimeVue from 'primevue/config';

wrapper = mount(DataTable, {
            global: {
                plugins: [PrimeVue]
            }
 });
```

**Reply by ambashy** [reply]
Thanks for the response . 
This i have done , But how to pass the mock data to the table.  That means how to fill the (:value) in a datatable component . Currently it is setting in the onmounted function in a component.
